src/storage/compactor/src/server.rs

```diff
@@ -76,7 +76,7 @@ pub async fn compactor_serve(
     ));
 
     // TODO: remove it after we can configure compactor independently.
-    config.storage.meta_cache_capacity_mb = config.storage.block_cache_capacity_mb;
+    config.storage.meta_cache_capacity_mb += config.storage.block_cache_capacity_mb;
```

Comment:
This looks a bit tricky. I guess this is because we need to make sure the meta cache is large enough to hold the prefetched SST blocks. Is this a permanant change or a temporary change?

Reply:
https://github.com/singularity-data/risingwave/pull/3268 will use an independent configuration for compactor. In the future, we will use another memory controller tablestore to replace `SstableStore` so that we do not need meta cache

Reply:
> #3268 will use an independent configuration for compactor. In the future, we will use another memory controller tablestore to replace `SstableStore` so that we do not need meta cache

Ok. Makes sense.